tests/infrastructure/test_scheduled_job_config_reader.py

```diff
@@ -0,0 +1,108 @@
+"""Tests scheduled job config reader."""
+
+import re
+import tempfile
+import textwrap
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+
+from sds_data_manager.constructs import scheduled_job_config_reader
+
+
+def test_scheduled_job_config_reader():
```

Comment:
You can let pytest take care of the cleanup for you with their provided fixture: https://docs.pytest.org/en/stable/how-to/tmp_path.html
```suggestion
def test_scheduled_job_config_reader(tmp_path):
```